Missing header unistd.h in lcm_logger.c

```
  CC     lcm_logger.o
lcm_logger.c: In function ‘write_thread’:
lcm_logger.c:261:13: warning: implicit declaration of function ‘fdatasync’ [-Wimplicit-function-declaration]
             fdatasync(fileno(logger->log->f));
             ^
```

--- a/lcm-logger/lcm_logger.c
+++ b/lcm-logger/lcm_logger.c
@@ -20,6 +20,8 @@
 #ifdef WIN32
 #define __STDC_FORMAT_MACROS            // Enable integer types
 #include <lcm/windows/WinPorting.h>
+#else
+#include <unistd.h> /* fdatasync */
 #endif
 
 #include <inttypes.h>

